testing/py-anyjson/APKBUILD

```diff
@@ -1,22 +1,28 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-anyjson
 _pkgname=anyjson
 pkgver=0.3.3
-pkgrel=1
+pkgrel=2
 pkgdesc="Wraps the best available JSON implementation available in a common interface"
 url="https://bitbucket.org/runeh/anyjson/"
 arch="noarch"
 license="BSD"
-makedepends="python2-dev python3-dev py-setuptools"
+makedepends="python2-dev python3-dev py-setuptools py-future py-nose"
```

Comment:
actually this changes are for `checkdepends` so like
```
checkdepends="py-future py-nose"
makedepends="python2-dev python3-dev py-setuptools"
```

Reply:
Didn't know about that, thanks!